mean_differences_mc/calculate_mean_difference.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from mean_differences_mc import *
+
+
+def calculate_mean_difference(minuend, subtrahend):
+    mean_minuend = np.mean(minuend)
+    mean_subtrahend = np.mean(subtrahend)
+    mean_diference = mean_minuend - mean_subtrahend
+    return mean_diference
+
+
+def calculate_mean_switched_difference(minuend, subtrahend, switches):
+    max_lim = len(minuend) - 1
+    minuend_switched = minuend
+    subtrahend_switched = subtrahend
+    mean_switched_difference = []
+    for i_switch in range(switches):
```

Comment:
Parece que no usamos la variable `i_switch`
```suggestion
    for _ in range(switches):
```